modules/apps/static/portal-osgi-web/portal-osgi-web-wab-generator-impl/src/main/java/com/liferay/portal/osgi/web/wab/generator/internal/WabGenerator.java

```diff
@@ -235,9 +233,4 @@ private void _registerURLStreamHandlerService(BundleContext bundleContext) {
 
 	private ServiceRegistration<FileInstaller> _serviceRegistration;
 
-	@Reference(
-		target = "(&(original.bean=true)(bean.id=javax.servlet.ServletContext))"
-	)
-	private ServletContext _servletContext;
```

Comment:
@tinatian @shuyangzhou Do we need to remove reference on `ServletContext`? Can they be kept just like `ModuleServiceLifecycle` or `Release`?

Also, since such usage is explicitly declaring a dependency, currently in this PR the reference is replaced with another service tracker. Is it really worthy?

Reply:
A similar case: https://github.com/liferay-core-infra/liferay-portal/pull/6915/commits/86784be3bded60d21c277f3a603cb3f3ceee5fd4